### ISSUE TITLE
Cannot delay video-only recording

### ISSUE THREAD

**Issue by stonebeach514** [collab-project/videojs-record#540]
Hi.
I tried a way to delay video-only recording but couldn't.I want to delay recording video-only by 5 seconds.I knew I would edit the record-toggle.js if it was audio only .
Which file do you edit for video only recoeding?

**Comment by thijstriemstra** [comment]
> I knew I would edit the record-toggle.js if it was audio only .

What line would you edit?

**Comment by stonebeach514** [comment]
I know that if the delay is audio only, edit line 72  of recorde-toggle.js as follows .

    handleClick(event) {
        let recorder = this.player_.record();
        if (!recorder.isRecording()) {
            recorder.start();
        } else {
            recorder.stop();
        }
    }

But I don't know where to edit for video-only :'-(

**Comment by thijstriemstra** [comment]
> I want to delay recording video-only by 5 seconds.

What is your use-case for this? Trying to understand why you'd need this.

**Comment by alberttjc** [comment]
> What is your use-case for this? Trying to understand why you'd need this.

I assume he wants to have a 5-second timer before the recording starts.

@stonebeach514 I believe it is also the same file but I had to edit the record-toggle.js in the dist folder. You can check out #348  

@thijstriemstra In saying that, if we edit the src file, should we re-build it? Apologies I'm fairly new to javascript! :(

**Comment by thijstriemstra** [comment]
> In saying that, if we edit the src file, should we re-build it? Apologies I'm fairly new to javascript! :(

@alberttjc yes, see https://collab-project.github.io/videojs-record/#/development

**Comment by alberttjc** [comment]
@thijstriemstra yup, cheers mate!
